recipes/sentry-breakpad/all/conanfile.py

```diff
@@ -0,0 +1,143 @@
+            "src/client/linux/minidump_writer/minidump_writer.cc",
+            "src/client/linux/minidump_writer/linux_ptrace_dumper.cc",
+            "src/client/linux/minidump_writer/cpu_set.h",
+            "src/client/linux/minidump_writer/directory_reader.h",
+            "src/client/linux/minidump_writer/line_reader.h"
+        ]
+
+        for file in files_to_patch:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "external", "breakpad", file),
+                "#include \"third_party/lss/linux_syscall_support.h\"",
+                "#include <linux_syscall_support.h>"
+            )
+
+        tools.save(os.path.join(self._source_subfolder, "external", "CMakeLists.txt"),
+                   textwrap.dedent("""\
+                    install(TARGETS breakpad_client
+                        ARCHIVE DESTINATION lib
+                        LIBRARY DESTINATION lib
+                        RUNTIME DESTINATION bin
+                    )
+                    file(GLOB COMMON_FILES breakpad/src/common/*.h)
+                    install(FILES ${COMMON_FILES}
+                        DESTINATION include/breakpad/common
+                    )
+                    set(PLATFORM_FOLDER)
+                    if(IOS)
+                        set(PLATFORM_FOLDER ios)
+                    elseif(APPLE)
+                        set(PLATFORM_FOLDER mac)
+                    elseif(UNIX)
+                        set(PLATFORM_FOLDER linux)
+                    endif()
+                    if(WIN32)
+                        set(PLATFORM_FOLDER windows)
+                    endif()
+                    if(NOT PLATFORM_FOLDER)
+                        message(FATAL_ERROR "Unknown os -> don't know how to install headers")
+                    endif()
+                    file(GLOB COMMON_PLATFORM_HEADERS breakpad/src/common/${PLATFORM_FOLDER}/*.h)
+                    install(FILES ${COMMON_PLATFORM_HEADERS}
+                        DESTINATION include/breakpad/common/${PLATFORM_FOLDER})
+                    install(DIRECTORY breakpad/src/client/${PLATFORM_FOLDER}
+                        DESTINATION include/breakpad/client
+                        FILES_MATCHING PATTERN *.h
+                    )
+                    install(DIRECTORY breakpad/src/google_breakpad/common
+                        DESTINATION include/breakpad/google_breakpad
+                        FILES_MATCHING PATTERN *.h
+                    )
+                   """), append=True)
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=os.path.join(self._source_subfolder, "external", "breakpad"))
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.names["pkg_config"] = "breakpad-client"
```

Comment:
```suggestion
        self.cpp_info.names["pkg_config"] = "breakpad-client"
        self.cpp_info.names["cmake_find_package"] = "breakpad"
        self.cpp_info.names["cmake_find_package_multi"] = "breakpad"
```
So the package is a drop in replacement to google/breakpad https://github.com/conan-io/conan-center-index/pull/5639

Reply:
Is that the provides attribute?

Reply:
I mean from CMake's perspective. With this change you can have have `find_package(breakpad)` regardless of the breakpad package you're using.

Reply:
I think the `provides` only prevents two packages with the same libraries/APIs from colliding.

Reply:
@prince-chrismc 
This is done to switch between `breakpad` and `sentry-breakpad` seamlessly.
`sentry-breakpad` is a forked version that, so far I know, don't has any intention to become standalone.

Reply:
I think it doesn't hurt to have it even if we're not planning on someone using it as a standalone version.
I can totally see someone trying to do it since the official version only builds on Linux.